python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -492,14 +491,13 @@ class RandomForestRegressor(Base):
            to a shared file. Cuml issue #1854 has been created to track this.
     """
     def _tl_model_handles(self, model_bytes):
-        task_category = REGRESSION_MODEL
```

Comment:
passing integers directly to c++ functions is not a good idea. We defined an enum for task_category. Using the enum gives a clearer picture to developers and users reading the code.

Reply:
since the enum does not work for classification, should we retain the only value left out of the enum?

Reply:
also, how did it work using CLASSIFICATION_MODEL before?

Reply:
since we supported fil for only binary classification, we would check the number of classes before building the treelite model and we would pass the task category enum `CLASSIFICATION_MODEL` which was set as 2. 
We can retain it out of the enum.